.circleci/config.yml

```diff
@@ -1,4 +1,6 @@
-version: 2
+version: 2.1
+orbs:
+  ship: auth0/ship@dev:a3ce828
```

Comment:
Targetting this commit: https://github.com/auth0/ship-orb/commit/a3ce828256173ad54f0fa4a5476ef49c1eff0ca0